semanticdb/metai/src/main/scala/scala/meta/metai/Result.scala

```diff
@@ -0,0 +1,13 @@
+package scala.meta.metai
+
+import scala.meta.io.AbsolutePath
+
+final class Result private (val status: Map[AbsolutePath, Boolean]) {
+  def success: Boolean = status.forall(_._2)
```

Comment:
Nitpick: I would name this `def isSuccess`

I try to start method names with `isX` when possible if they return boolean responses.

Reply:
Will fix.